ci-operator/step-registry/baremetalds/packet/setup/baremetalds-packet-setup-commands.sh

```diff
@@ -18,6 +18,7 @@ cat > packet-setup.yaml <<-EOF
   vars:
     - cluster_type: "{{ lookup('env', 'CLUSTER_TYPE') }}"
     - slackhook_path: "{{ lookup('env', 'CLUSTER_PROFILE_DIR') }}"
+    - packet_hostname: "{{ packet_hostname }}-{{ ansible_date_time.iso8601_basic_short }}" # Concatenate the current timestamp to packet_hostname
```

Comment:
Please add also the teardown otherwise we'll get a leak even in the rehearse job.
Note: we should print the hostname as right now it's not complete when shown in the name of the play

Reply:
If the tests pass we should be golden :)